README.md

```diff
@@ -1,4 +1,4 @@
-pvlib-python
+[pvlib-python](docs/sphinx/source/_images/pvlib_logo_horiz.png)
```

Comment:
need to add preceding exclamation mark for image to render, see [GitHub Markdown guide](https://guides.github.com/features/mastering-markdown/#)

    ![pvlib-python](docs/sphinx/source/_images/pvlib_logo_horiz.png)

Reply:
Thanks.  Do you know how to specify a display size?

I can make a pre-release tag that will build on readthedocs.  Otherwise I don't know how to view the change to index.rst

Reply:
You can go into your readthedocs account and set up a new project that points to your fork. cwhanse-pvlib-python or something like that. Then you have to go into your new project's settings and turn on the build for your branch. 

Alternatively, I can test locally if we're ready for that.

Reply:
I set up a readthedocs for my fork, that is the long-term solution.

Reply:
The `index.rst` files renders in the viewer here. 

Ready for review.